types/src/params.rs

```diff
@@ -176,18 +176,14 @@ impl<'a> ParamsSequence<'a> {
 		T: Deserialize<'a>,
 	{
 		let mut json = self.0;
-		tracing::trace!("[next_inner] Params JSON: {:?}", json);
 		match json.as_bytes().first()? {
 			b']' => {
 				self.0 = "";
-
-				tracing::trace!("[next_inner] Reached end of sequence.");
```

Comment:
removed these, I'm quite confident that this works as intended and these trace logs are a bit extreme :)